### DISCUSSION TITLE
Error "Route [filament.auth.login] not defined" after disabling login

### DISCUSSION THREAD

**Discussion by sh4msi** [filamentphp/filament#4181] [Help]
Hi,
When I disable the default filament login, I encounter the error Route [filament.auth.login] not defined
I cleared the cache but the problem still persists

Currently, the only solution I found is redirecting to the desired page

Is there a better solution?

**Comment by danharrin** [comment]
No, redirecting to another page is correct 👍 We need to know where to send users to login
